src/test/java/redis/clients/jedis/tests/ProtocolTest.java

```diff
@@ -119,4 +120,17 @@ public void nullMultiBulkReply() {
     List<String> response = (List<String>) Protocol.read(new RedisInputStream(is));
     assertNull(response);
   }
-}
\ No newline at end of file
+
+  @Test
```

Comment:
I didn't use the `expected` annotation parameter because it wouldn't allow to [assert the correct content](https://github.com/xetorthio/jedis/pull/411/files/f399e4fa213fe298e26613d3c03055ab75722d3b#diff-43462f06c65dec4783b0c0abcb502e15R131) of the exception's message.

Do you imply that it should not be tested?